pkg/configmap/configmap.go

```diff
@@ -0,0 +1,53 @@
+package configmap
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+	corev1 "k8s.io/api/core/v1"
+	kubetypes "k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// OperatorConfigMapName holds the name of configmap
+const (
+	OperatorConfigMapName      = "gcp-project-operator"
+	operatorConfigMapNamespace = "gcp-project-operator"
+)
+
+// OperatorConfigMap store data for the specified configmap
+type OperatorConfigMap struct {
+	BillingAccount string `mapstructure:"billingAccount"`
+	ParentFolderID string `mapstructure:"parentFolderID"`
+}
+
+// CheckValueNotExist checks if OperatorConfigMap filled properly
+func CheckValueNotExist(configmap OperatorConfigMap) error {
```

Comment:
I find the name of this function confusing it sounds like it checks if a value exists (or does not exist?), I suggest something like ValidateConfigMap, which would make clearer that it verifies all the expected values exist.

Reply:
`ValidateConfigMap` looks good.